lib/logstash/inputs/redis.rb

```diff
@@ -201,13 +201,13 @@ def queue_event(msg, output_queue)
   end
 
   # private
-  def shutting_down?
-    @shutdown_requested
+  def stop?
+    @stop_requested
```

Comment:
This will be provided by the base class when using the stop method. Check https://github.com/elastic/logstash/pull/3895/files#diff-703bc5e68070272130474602611cf11eR114 for details.